docs/pages/beta/crm-beta/api-reference/get-all-contacts.mdx

```diff
@@ -13,6 +13,16 @@ curl --request POST \
   --data '{
   "include": { $native: false },
   "cursor": "123123",
+  "filter": {
```

Comment:
ah hahah, tiny thing but can we actually keep this the same as the filter example in the yaml, otherwise the generators will fail rip